packages/library-legacy/test/connection.test.ts

```diff
@@ -4919,90 +4919,61 @@ describe('Connection', function () {
       const connection = new Connection(url, 'confirmed');
       const newAccount = PublicKey.unique();
 
-      let payerKeypair = new Keypair();
-      let testTokenMintPubkey: PublicKey;
-      let testOwnerKeypair: Keypair;
-      let testTokenAccountPubkey: PublicKey;
-      let testSignature: TransactionSignature;
+      const payerPublicKey = new PublicKey(
+        '6bffFjUMnnk3tXFJAS8pp9tz6m7oJAd27vm9rKqnLtVv',
+      );
```

Comment:
Is this even needed anymore? Looks like no.

Reply:
It's used in one dumb place and I was too lazy to see if I could replace it. Let me see.

Reply:
Gone!